src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -105,16 +105,28 @@ def generate_MCP_1_default_config(self, filename):
         dpa_3 = json.load(
           open(os.path.join('testcases', 'testdata', 'dpa_2.json')))
 
-        # Registration configuration for iteration 1
-        cbsd_records_iter_1 = {'registrationRequests': [device_1, device_2],
-                               'grantRequests': [grant_request_1, grant_request_2],
-                               'conditionalRequests': conditionals_iter_1
-                               }
-        cbsd_records_iter_2 = {'registrationRequests': [device_3, device_4],
+        # Registration and grant records for multiple iterations
+        cbsd_records_iter_1_domain_proxy_0 = {'registrationRequests': [device_1, device_2],
+                                              'grantRequests': [grant_request_1, grant_request_2],
+                                              'conditionalRegistrationData': conditionals_iter_1
```

Comment:
Conditional registration data is injected, it does not go through a DP.

Reply:
Agreed,   conditional data is injected through Admin interface, we kept this info along   with DP, so that we can easily pass the info to domain proxy object, which   will in turn take care of injecting the conditional registration data

Reply:
I don't think the domain proxy object can inject the conditional registration data since this is done through the admin interface.

Reply:
Sure,   will ensure that the conditional registration data  will be injected through admin interface   before passing the registration data to domain proxy object in the test case